src/libs/extra/metadata.liq

```diff
@@ -0,0 +1,99 @@
+# Store and retrieve file covers using metadata. This returns a set of
+# getter/setter methods that can be used to store and retrieve cover art.
+# Typical usage is to set cover art in a `on_metadata` handler and retrieve
+# it in a `video.add_image` operator.
```

Comment:
Since this can be a bit complicated for some users, it would be nice to add another `source.video_from_cover` operator based on this which takes an audio track and adds a video track based on the cover!